src/CairoMakie.jl

```diff
@@ -531,15 +531,41 @@ function draw_plot(scene::Scene, screen::CairoScreen, primitive::Combined)
 end
 
 function draw_plot(screen::CairoScreen, scene::Scene)
+
+    # get the parent area to correct for its pixel size when translating
+    parent_area = if isnothing(scene.parent)
+        pixelarea(scene)[]
+    else
+        pixelarea(scene.parent)[]
+    end
```

Comment:
```suggestion
    parent_area = pixelarea(root(scene))[]
```
if you want the full thing

Reply:
not sure, do we need the parent or the root? This would only matter for scenes that are child scenes of child scenes

Reply:
right, but we should at least handle it (consider an animation of multiple scenes with a popup)

Reply:
plus, if Cairo is using root-level coordinates, we would need to as well

Reply:
Hm, although, we should just use the rect of the cairoscreen, no? Thats what we're trying to draw after all

Reply:
Good point - we can access that by:
```suggestion
    parent_area = round.(Int, (screen.surface.width, screen.surface.height)
```

Reply:
actually, scrap this